numpy/lib/function_base.py

```diff
@@ -84,11 +84,15 @@ def histogram(a, bins=10, range=None, normed=False, weights=None,
     ----------
     a : array_like
         Input data. The histogram is computed over the flattened array.
-    bins : int or sequence of scalars, optional
+    bins : int or sequence of scalars, str, optional
         If `bins` is an int, it defines the number of equal-width
         bins in the given range (10, by default). If `bins` is a sequence,
         it defines the bin edges, including the rightmost edge, allowing
         for non-uniform bin widths.
+        If `bins` is a string, it selects an automatic bin estimator, either
+        ["fd", "scott", "sturges", "rice", "auto"]. If no match found,
+        it reverts to the auto method. For visualisation libraries, we suggest
```

Comment:
If no match is found then raising a ValueError is preferable. Most likely the user made a typo.